01_clean_code/1_long_code_is_not_good_code/1_clean_code_part_1.md

```diff
@@ -22,33 +22,6 @@ After this lesson you should:
 - Learn how to start refactoring your current codebase,
 - Antipatterns to avoid.
 
-## Uncle Bob - Clean code lesson 2 (1:06)
-
-[Link to video](https://www.youtube.com/watch?v=2a_ytyt9sf8)
-
-In the second part of the "Coding Better World Together", Uncle Bob focuses on the clean code rules for comments and rules to write names.
-
-After this lesson you should:
-
-- Understand what kinds of comments to write,
-- Know about _lying comments_ and when _not_ to write comments,
-- Be able to reveal your intent while naming things.
-
-## Python Type Checking (Guide) (1:40)
-
-![RealPython cover](../images/4b3cc00ea7b463fb46c2dfbe07beab3d6e708384e6938b82cc3e4f65767da9e2.png)  
-
-[Link to RealPython guide](https://realpython.com/python-type-checking/)
-
-This is a very complete guide on Python type hints from _RealPython_. It's a comprehensive guide that covers a lot of ground and a good piece to keep for future reference.
-Type hints are optional in Python, but using them will allow you to catch bugs before they happen, and allow readers to understand your classes and functions just by looking at their signatures.
-
-After this lesson you should:
-
-- Learn how to add type hints to your code,
-- Be able to create type aliases,
-- Know how to run a static type checker like `mypy`.
-
 ## 7 Python Code Smells (0:22)
 
 ![Arjan Codes YouTube cover](../images/186f8c6d5b3ee296613d84f8b1b5a304ca39034a513d79bcd09688ef91f8b6cb.png)
```

Comment:
the links are not rendering in the preview:

<img width="412" alt="image" src="https://github.com/DareData/lp-foundations/assets/424192/029034db-e639-4ae6-aa0f-9e2a9c12fc46">

Reply:
Fixed!